src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-fri calendar--length-30">
+      <div class="calendar__day"></div>
```

Comment:
Modifiers should be written with two dashes by BEM. Like` calendar--start-day--fri`

Reply:
`calendar--start-day--fri` isn't a correct modifier, with this modifier you created modifier `--fri` from `--start-day` modifier, according to bem you can't create modifier from another modifier